Scripts/script-GetTime.yml

```diff
@@ -39,6 +39,29 @@ script: |-
           case 'Date':
               timeStr = d.toDateString();
               break;
+        case 'year':
```

Comment:
there seems to be indentation between the case lines.. can you fix that?

Reply:
@AndyDemisto ??